uint16 stabilized_time
float32 yaw_current
float32 pitch_current
float32 roll_current
float32 depth_current
float32 temperature
uint16[6] thruster_values
